src/App.tsx

```diff
@@ -11,23 +12,23 @@ const App = () => {
 	return (
```

Comment:
If you are using TS you have to add types to almost everything you have.
For components I suggest for you to use `React.FC<PropsTypes>`. for useState you can do it like this: `const [a, b] = useState<type>(...)

For handlers that don't return anything you need to write `const handleAddNewCourseButtonClick = (): void => setAddingMode(true);`